### PR TITLE
Use new zowe-install-xmem.sh script

### PR DESCRIPTION
**Pull request by John-A-Davies**
Test the new zowe-install-xmem.sh script which copies the xmem server PROCLIB, PARMLIB and LOADLIB to the live datasets.  Tests the fix for zowe-install-packaging issue #914

### CODE REVIEW

File: scripts/install-zowe.sh [hunk 1]
```diff
@@ -493,12 +541,49 @@ else #not SMPE
...
+  EXIT_CODE=$?
+  if [[ "$EXIT_CODE" != "0" ]]; then
+    echo "[${SCRIPT_NAME}][error] ${RUN_SCRIPT} failed."
+    echo
+    exit 1
+  else
+    echo "[${SCRIPT_NAME}] ${RUN_SCRIPT} succeeds."
+    echo
+  fi
+fi #End SMPE if
```
Comment (stevenhorsman) [inline scripts/install-zowe.sh]: Please revert/remove the duplicated code and restore to after the if block

File: scripts/install-zowe.sh [hunk 2]
```diff
@@ -377,6 +395,36 @@ if [[ "$CI_IS_SMPE" = "yes" ]]; then
...
   ${CIZT_INSTALL_DIR}/install-xmem-server.sh
+
+  # Copy xmem server PROCLIB, PARMLIB and LOADLIB members into targets
+  echo "calling zowe-install-xmem.sh with"
+  echo "  datasetprefix ${DATA_SET_PREFIX}"
+  echo "  zss loadlib   ${CIZT_ZSS_LOADLIB_DS_NAME}"
+  echo "  zss parmlib   ${CIZT_ZSS_PARMLIB_DS_NAME}"
+  echo "  zss proclib   ${CIZT_ZSS_PROCLIB_DS_NAME}"
+  cd $CIZT_ZOWE_ROOT_DIR/scripts/utils
+  chmod a+x ./zowe-install-xmem.sh
```
Comment (stevenhorsman) [inline scripts/install-zowe.sh]: Please remove this line - if the script doesn't have execute permission, we need to know about it as a install defect

File: scripts/install-zowe.sh [hunk 3]
```diff
@@ -493,13 +500,48 @@ else #not SMPE
...
 fi
 
+echo "calling zowe-install-xmem.sh with"
+echo "  datasetprefix ${DATA_SET_PREFIX}"
+echo "  zss loadlib   ${CIZT_ZSS_LOADLIB_DS_NAME}"
+echo "  zss parmlib   ${CIZT_ZSS_PARMLIB_DS_NAME}"
+echo "  zss proclib   ${CIZT_ZSS_PROCLIB_DS_NAME}"
+
+ls -l     ./zowe-install-xmem.sh # check
+chmod a+x ./zowe-install-xmem.sh
```
Comment (stevenhorsman) [inline scripts/install-zowe.sh]: Please remove this line - if the script doesn't have execute permission, we need to know about it as a install defect

**Comment by Joe-Winchester** [comment]
https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-144/12/ is green to merging this and deleting the branch.  Tks @John-A-Davies and @stevenhorsman for getting this through.  Coffee/tea on me tomorrow morning !

**Pull request merged**